lib/stripe/stripe_object.rb

```diff
@@ -43,7 +43,7 @@ def inspect
 
     def refresh_from(values, opts, partial=false)
       @opts = opts
-      @previous_metadata = values[:metadata]
+      @previous = values
```

Comment:
Also, I think it'd be best to do a deep copy here. Otherwise, if someone modifies `values` after calling `refresh_from`, it's going to do the wrong thing. `Marshal.load(Marshal.dump(values))` should do the trick.